specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/examples/compute/DedicatedHostRestart.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2021-07-01",
```

Comment:
pls. update the api version to 2021-11-01

Reply:
will update, thanks